backend/src/services/org-admin/org-admin-service.ts

```diff
@@ -144,30 +139,6 @@ export const orgAdminServiceFactory = ({
       });
     }
 
-    const botPrivateKey = crypto
```

Comment:
question: these do not need to be touched?

<img width="1067" height="608" alt="image" src="https://github.com/user-attachments/assets/47fc1c12-4bc5-4b07-98da-ff8f2df7f173" />